_tech-blog/2020-08-31-gsoc-20-resume.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: GSoC 20 - Automatized support for Organised Editing Guidelines
+date: 2020-08-31 12:30:00 Z
+Summary Text: Here is the resume from the GSoC' 20 project that adds
+  automatized support for Organised Editing Guidelines in
+  Tasking Manager.
+Feature Image: "/uploads/gsoc-20-architecture.jpg"
+Person:
+- Wille Marcel
```

Comment:
add your name here

Reply:
and remove my name 😄

Reply:
Alright, updated to my name.
I used your name because I thought it was necessary to be a member of the tech team smile